builder/bundles.go

```diff
@@ -452,7 +392,15 @@ func resolvePackages(numWorkers int, set bundleSet, packagerCmd []string, emptyD
 
 			bundleRepoPkgs.Store(bundle.Name, rpm)
```

Comment:
Are we are using bundleRepoPkgs anywhere? I think we can skip creating it.

Reply:
ohk, we are using it during build validation.

Reply:
we can still populate the bundleRepoPkgs only for validation step and save on memory for normal builds.

Reply:
@jwakre ??

Reply:
I will make a small, quick change to so that bundleRepoPkgs is not populated when calling `mixer build bundles`.